porcupine/plugins/urls.py

```diff
@@ -28,6 +28,7 @@ def find_urls(text: tkinter.Text, start: str, end: str) -> Iterable[Tuple[str, s
         url = url.split(" ")[0]
         url = url.split("'")[0]
         url = url.split('"')[0]
+        url = url.split('`')[0]
```

Comment:
Black likes double quotes (could also run `black urls.py`):

```suggestion
        url = url.split("`")[0]
```

Reply:
Thanks. I know it's a bad practice but I normally use single quote because I like it better. Fixed.

Reply:
It's not "bad practice", it's just not how black does it. Black is very opinionated by design, and there's lots of things that aren't bad but it changes anyway.